heartpy/heartpy.py

```diff
@@ -294,6 +295,23 @@ def process(hrdata, sample_rate, windowsize=0.75, report_time=False,
                                          desired_sample_rate=high_precision_fs, working_data=working_data)
 
     working_data = calc_rr(working_data['peaklist'], sample_rate, working_data=working_data)
+    
+    # Obtain Mode
+    mode_v = mode(working_data['RR_list'])
+    measures['Mo'] = mode_v
+
+    # Obtain VR (variational range)
+    vr = np.max(working_data['RR_list']) - np.min(working_data['RR_list'])
+    measures['VR'] = vr
+
+    # Obtain AMo (mode amplitude %)
+    num_mode = np.count_nonzero(working_data['RR_list'] == mode_v)
```

Comment:
Technically this is correct, but it depends on the type of input data. The RR Intervals have to be bucketed into 50ms buckets according to Kubios and the paper by Baevsky (https://cardiometry.net/media/eng/issues/no10-may-2017/heart-rate-variability-analysis.pdf).

Taking a strict equality sounds wrong to me.